interface/therapy_groups/therapy_groups_views/listGroups.php

```diff
@@ -0,0 +1,411 @@
+                "info": '<?php echo xlt("Showing page") .' _PAGE_ '. xlt("of") . ' _PAGES_'; ?>',
+                "infoEmpty": '<?php echo xlt("No records available") ?>',
+                "infoFiltered": '<?php echo "(" . xlt("filtered from") . ' _MAX_ '. xlt("total records") . ")"; ?>',
+                "infoPostFix":  "",
+                "search":       "<?php echo xlt('Search')?>",
+                "url":          "",
+                "oPaginate": {
+                    "sFirst":    "<?php echo xlt('First')?>",
+                    "sPrevious": "<?php echo xlt('Previous')?>",
+                    "sNext":     "<?php echo xlt('Next')?>",
+                    "sLast":     "<?php echo xlt('Last')?>"
+                }
+            },
+            initComplete: function () {
+                $('#therapy_groups_list_filter').hide(); //hide searchbar
+            }
+        });
+
+        /* Order by Start Date column (descending) */
+        table.order( [ 4, 'desc' ] ).draw();
+
+        /* Hide/Show filters */
+        $("#show_filters").click(function () {
+            $('#filters').show();
+            $("#hide_filters").show();
+            $("#show_filters").hide();
+
+        });
+        $("#hide_filters").click(function () {
+            $('#filters').hide();
+            $("#hide_filters").hide();
+            $("#show_filters").show();
+        });
+
+        /* ------------ Toggle filter functions on keyup/change ----------- */
+
+        /*
+        * Note: where there is an explicit extension made for the filter, just table.draw() was used.
+        * Otherwise 'table.columns(  ).search( this.value ).draw();' was used.
+        */
+
+
+        /* ---- Datetimepickers ---- */
+        $('#group_from_start_date_filter').change( function() {
+            table.draw();
+        } );
+        $('#group_to_start_date_filter').change( function() {
+            table.draw();
+        } );
+
+        $('#group_from_end_date_filter').change( function() {
+            table.draw();
+        } );
+        $('#group_to_end_date_filter').change( function() {
+            table.draw();
+        } );
+
+        /* --- Text inputs --- */
+        $('#group_name_filter').keyup( function() {
+            table.draw();
+        } );
+        $('#group_id_filter').keyup( function() {
+            table.draw();
+        } );
+
+        /* ---- Select Boxes ---- */
+        $('#group_type_filter').change(function () {
+            table.columns( 2 ).search( this.value ).draw();
+        } );
+
+        $('#group_status_filter').change( function() {
+            table.draw();
+        } );
+
+        $('#counselors_filter').change( function() {
+            table.columns( 6 ).search( this.value ).draw();
+        } );
+
+        /* ----------------- End of filter toggles -------------------- */
+
+
+        /* --------- Reset Filters ------ */
+        $('#clear_filters').click(function(){
```

Comment:
Fixed. So just to clarify, whenever we load a web page using javascript we should first use this method?